### PR TITLE
NW 4 - Gulnihal Naldoken - HTML/CSS Week 2

### PR DESCRIPTION
**Pull request by gulnihaln**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,5 +15,73 @@
...
+                <div class="menu">
+                
+                        <a href="#"><span>Meet Karma</span></a>
+                        <a href="#">How it Works</a>
+                        <a href="#">Store</a>
+                        <a href="#">Blog</a>
+                        <a href="#">Help</a>
+                        <a href="#">Login</a>
+                    
+                </div>
```
Comment (zeinaji) [inline index.html:36]: Great work with making the navbar responsive! My only comment here would be to wrap it in a `<nav>` tag to make it more responsive

File: css/style.css [hunk 2]
```diff
@@ -16,4 +19,338 @@ body {
...
+
+
+@media only screen and (max-width: 768px) {
+    html {
+        width: 100%;
+        font-size 12px;
+    }
+
+    section.grid {
+        display: flex;
```
Comment (zeinaji) [inline css/style.css:293]: Having `.grid` here instead of `section.grid` would also work :)

**Review by zeinaji — COMMENTED** [review]
This looks exactly as it should, and nice work with the semantic HTML and making it responsive!

File: index.html [hunk 3]
```diff
@@ -15,5 +15,73 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <header class="header-container">
+        <div class="logo">
+            <img src="/img/karma-logo.svg" 
+            alt="karmalogo" />
```
[the remark this answers is not in the document]
Comment (gulnihaln) [reply index.html:21]: I think when I click on the backslash to choose images, it stays there. I've learned something thanks

File: index.html (hunk 1 above)
Comment (gulnihaln) [reply index.html:36]: It was <nav>, but a friend helped me to fix my hamburger menu, then she replaced some of my tags. I'll do it, thanks!

**Pull request closed without merging**